docs/src/openapi-v1.yaml

```diff
@@ -1342,6 +1342,54 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ParasLeaseInfo'
+  /paras/head/included-candidates:
+    get:
+      tags:
+      - paras
+      summary: |
+        Get the heads of all the current included candidate parachains at the most recent finalized head.
+      description: |
+        Returns an object with all the parachain id's as keys, and their headers as values.
+      parameters:
+      - name: at
+        in: query
+        description: Block at which to retrieve para's leases at.
+        required: false
+        schema:
+          type: string
+          description: Block identifier, as the block height or block hash.
+          format: unsignedInteger or $hex
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ParasHeaders'
+  /paras/head/backed-candidates:
+    get:
+      tags:
+      - paras
+      summary: |
+        Get the heads of all the current backed candidate parachains at the most recent finalized head.
```

Comment:
Similarly as above:

I would write here:

"Get the heads of the backed parachain candidates at the specified block height or at the most recent finalized head otherwise."

With this endpoint we are pulling the heads of the candidates that are backed at a given relay chain block, but this does not imply we are retrieving all the currently backed (but not yet available) candidates at this height.

Reply:
Actually, there is a storage entry `paraInclusion.pendingAvailability` which stores exactly this, all the candidates that have been backed so far but they have not been included yet (availability step).

Reply:
Okay cool, I updated the description. 

> paraInclusion.pendingAvailability

Is this something you would like to see included, maybe in a different endpoint?

Reply:
Perhaps this won't be as interesting/useful as the two endpoints here but it is something to consider for sure. Perhaps a lower priority task for the incoming months?

Reply:
> Perhaps a lower priority task for the incoming months?

Yea I agree with this. Im happy to tackle this again, as time goes on. Would be good to see if this will have merit among many users.